pkg/api/types.go

```diff
@@ -2262,6 +2262,11 @@ const (
 	// Number of Pods that may be running on this Node: see ResourcePods
 )
 
+const (
+	// Namespace prefix for opaque counted resources (alpha).
+	ResourceOpaqueIntPrefix = "opaque-int.alpha.kubernetes.io/"
```

Comment:
patterns are fuzzy, but the closest thing I see is AREA.alpha.kubernetes.io/FEATURE  

Where area = things like control-plane/scheduler/pod 

I debate if this is pod?  really seems like pod to me.